lib/src/builder/codegen/typed_map_impl_generator.dart

```diff
@@ -390,11 +390,14 @@ class _TypedMapImplGenerator extends TypedMapImplGenerator {
 
   String _generateUiFactoryConfig(FactoryNames factoryName) {
     return 'final UiFactoryConfig<${names.implName}> '
-        '${factoryName.configName} = UiFactoryConfig(\n'
+        '${factoryName.privateConfigName} = UiFactoryConfig(\n'
         'propsFactory: PropsFactory(\n'
         'map: (map) => ${names.implName}(map),\n'
         'jsMap: (map) => ${names.jsMapImplName}(map),),\n'
-        'displayName: \'${factoryName.consumerName}\');\n\n';
+        'displayName: \'${factoryName.consumerName}\');\n\n'
+        '@Deprecated(\'Use `_\\${factoryName.publicConfigName}` instead.\')\n'
```

Comment:
I feel like it'd be useful to add more context here. Just something like (feel free to change my suggestion to anything that makes more sense to you though 🤷‍♂️ ):
```suggestion
        '@Deprecated(\'This member is incompatible with Dart >2.9.0 and should not be used. Use `_\\${factoryName.publicConfigName}` instead.\')\n'
```

Reply:
Updated in 8ce88ad !